.stylelintrc

```diff
@@ -47,13 +46,10 @@
     "media-query-list-comma-newline-before": "never-multi-line",
     "media-query-list-comma-space-after": "always-single-line",
     "media-query-list-comma-space-before": "never",
-    "media-query-parentheses-space-inside": "always",
     "no-eol-whitespace": true,
     "no-missing-eof-newline": true,
     "number-leading-zero": "always",
-    "number-max-precision": 2,
     "number-no-trailing-zeros": true,
-    "number-zero-length-no-unit": true,
```

Comment:
It's because of the responsive, after a certain width some elements doesn't need `margin-top` anymore. Do you have some idea of what could we do?